pages/story/[id].tsx

```diff
@@ -81,7 +80,9 @@ export default function StoryPage(props: StoryPageProps): JSX.Element {
         <StoryDetail story={story} onClose={handleClose} />
       </Box>
       <FloatingRibbon>
-        <Button onClick={onOpen}>Share This Story</Button>
+        <Button onClick={onOpen} marginTop="5px" marginBottom="5px">
```

Comment:
Can also be simplified 
```suggestion
        <Button onClick={onOpen} my={"5px"}>
```